packages/nimble-components/src/tooltip/tests/tooltip.spec.ts

```diff
@@ -20,7 +28,11 @@ describe('Tooltip', () => {
         if (!iconElement) {
             return false;
         }
-        return window.getComputedStyle(iconElement).display !== 'none';
+        const display = window.getComputedStyle(iconElement).display;
+        expect(display).not.toBe('');
```

Comment:
We shouldn't include `expect` statements in these helper functions.
If there is some assertion that needs to be made in helper code then throw an exception instead.
expects should be in the top-level spec code, i.e. the `it` block

Reply:
Throwing error instead